traffic_ops/experimental/go-api/tenant/EndPointSeeder.go

```
@@ -0,0 +1,202 @@
+    params := mux.Vars(aRequest)
+
+    var id int
+    var name string
+    var active bool
+    var parent_id sql.NullInt64
+    var updated time.Time;
+    
+    err := aEndpointSeeder.myDb.QueryRow("SELECT * FROM tenant WHERE id="+params["id"]).Scan(&id, &name, &active, &parent_id, &updated)
+    if err == sql.ErrNoRows{//TODO make it work
+        aResponseWriter.WriteHeader(http.StatusNotFound)
+        fmt.Fprint(aResponseWriter, params["id"]+" not found")
+        return
+    }
+    aEndpointSeeder.checkErr(err)
+    
+    var parent_id1 int
+    if parent_id.Valid {
+        parent_id1 = int(parent_id.Int64)
+    } else {
+       parent_id1 = 0        
+    }
+
+    json.NewEncoder(aResponseWriter).Encode(Tenant{Id: id, Name: name, Active: active, ParentId: parent_id1})
+    return
+}
+    
+
+ 
+func (aEndpointSeeder *EndpointSeeder) getListEndpoint(aResponseWriter http.ResponseWriter, aRequest *http.Request) {
+    var tenants []Tenant
+    rows, err := aEndpointSeeder.myDb.Query("SELECT * FROM tenant")
+    aEndpointSeeder.checkErr(err)
+    
+    for rows.Next() {
+        var id int
+        var name string
+        var active bool
+        var parent_id sql.NullInt64
+        var updated time.Time;
+            
+        err = rows.Scan(&id, &name, &active, &parent_id, &updated)
+        aEndpointSeeder.checkErr(err)
+
+        var parent_id1 int
+        if parent_id.Valid {
+            parent_id1 = int(parent_id.Int64)
+        } else {
+            parent_id1 = 0        
+        }
+    
+        tenants = append(tenants, Tenant{Id: id, Name: name, Active: active, ParentId: parent_id1})
+    }
+    
+    json.NewEncoder(aResponseWriter).Encode(tenants)
+
+}
+ 
+func (aEndpointSeeder *EndpointSeeder) createEndpoint(aResponseWriter http.ResponseWriter, aRequest *http.Request) {
+    var tenant Tenant
+    _ = json.NewDecoder(aRequest.Body).Decode(&tenant)
```

This error needs checked. Ignoring errors is never ok in production code.